### DISCUSSION TITLE
Making the disk persistent with an installation wth "node" option

### DISCUSSION THREAD

**Discussion by bourbe** [FlowiseAI/Flowise#1157] [Q&A]
Hello,

I installed my flowise with render but I am strugling to make my disk persistent.

1. I followed the instruction in this video to make my apps persistent : https://youtu.be/Fxyc6-frgrI?t=579 but I got an error and it broke my installation
2. I did some research and I found this issue : https://github.com/FlowiseAI/Flowise/issues/289 and I followed the recommendations in this page: https://docs.flowiseai.com/deployment/render. As result I got an error and it broke my installation

So please, how can I make my disk persistent with an installation with the "node" option

Thanks for your answer, it is urgent for my client

Regards

**Comment by sdadesky** [comment]
Did you ever get an answer to this?
